docker/docker-compose.yml

```diff
@@ -36,10 +36,20 @@ services:
     ports:
       - 8086:8086
   grafana:
-    image: "grafana/grafana"
+    image: "imlteam/grafana:dev"
```

Comment:
remove dev tag

Reply:
Yes, this was intentional for the time being for testing. Once we are good with the changes my intention was change change the tag to 5.1. It will be the same for the patches on top of this as well.